docs/en/tech_notes.md

```diff
@@ -8,10 +8,6 @@ LDAP-only sequence:
 1. User fills in the credentials
 1. `LDAPAuthenticator::authenticate` is called
 1. Authentication against LDAP is performed by SilverStripe's backend.
-1. If `Member` record is not found, stub is created with some basic fields (i.e. GUID), but no group mapping.
-1. A login hook is triggered at `LDAPMemberExtension::memberLoggedIn`
```

Comment:
Keep this and change to `LDAPMemberExtension::afterMemberLoggedIn()`

Reply:
DONE